cmd/lxdk/create.go

```diff
@@ -293,6 +302,9 @@ func createContainers(state config.ClusterState, numWorkers int, is lxdclient.In
 	created := make(map[string][]string)
 	created["worker"] = []string{}
 	for _, image := range []string{"etcd", "controller", "worker", "registry"} {
+		if image == "registry" && !state.RegistryEnabled {
```

Comment:
Why do we do this test here?

Reply:
This just prevents the registry from being created if the user didn't enable it. Honestly this and a couple other places in the create logic are kind of trash from when the primary goal was just "get it working," but since we are planning on creating images at runtime later, I didn't think this PR was a good place to fix it.